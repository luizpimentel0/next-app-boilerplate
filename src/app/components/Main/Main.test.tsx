import React from 'react'
import { render, screen } from '@testing-library/react'
import { describe, expect, it } from 'vitest'
import { Main } from './index'

describe('Main component', () => {
  it('should render the heading', () => {
    const { container } = render(<Main />)

    expect(screen.getByRole('heading', { name: /oi/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
