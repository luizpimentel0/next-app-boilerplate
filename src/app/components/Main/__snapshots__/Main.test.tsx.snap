// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Main component > should render the heading 1`] = `
<main>
  <h1>
    Oi
  </h1>
</main>
`;
